apps/site/components/Blog/BlogHeader/index.tsx

```diff
@@ -21,6 +21,7 @@ const BlogHeader: FC<BlogHeaderProps> = ({ category }) => {
         {t('layouts.blog.title')}
         <Link
           href={`/feed/${currentFile}`}
+          locale="en"
```

Comment:
Is this the only place that needs update? I also wonder, should we really redirect people out of their current locale to the "en" one? Although yes, the content is in English, the rest of the site is still localized... So I'm unsure if this is a warranted solution that makes sense.

cc @nodejs/nodejs-website

Reply:
@ovflowd 
We had a discussion before
https://github.com/nodejs/nodejs.org/pull/8073#issuecomment-3190635849

BTW, the user will still stay in their current locale even though they click the RSS icon.
And it should be a better UX compare with the original one: go to 404 not found page without any explanation to the user.

@ovflowd Do you have a better solution in your mind ? we can have further discussion on that

Reply:
I'm unsure whatt his locale flag is really doing. I'll defer to @bmuenzenmeyer or the @nodejs/nodejs-website team to make an educated judgement here. Feel free to dismiss my review.

Reply:
`Previous behavio`r: For any locale other than en, when users clicked the RSS feed link, they were redirected to a 404 page. This was because, during the RSS feed route generation, we deliberately only generated the en route. (There was a commit [2c727554e4ba2facf8fd815fdeba9500c7297116] that attempted to change this behavior, but feedback suggested it should instead be handled via the link.)

`The new approach`: Regardless of the locale, clicking the RSS feed link will now direct users to the en feed. Since the team currently has no plans to translate the feed, this approach is better than the previous one where users would just encounter a 404 with no explanation.

@avivkeller Do you have the permission to bypass the request change ? Or what is the next step ?

Reply:
I see now, this is just a link to the RSS feed, not the blog post.